contracts/src/v0.8/functions/dev/1_0_0/interfaces/IFunctionsClient.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
+
+/**
+ * @title Chainlink Functions client interface.
+ */
```

Comment:
Can we remove this function? It shouldn't be necessary.

Else, if there is some reason to keep it, we should also have `getThresholdPublicKey` and `getPerNodePublicKeys`, right?